drop/setupDrop.py

```diff
@@ -24,14 +25,47 @@ def setupPaths(projectRoot):
     return repoPaths, projectPaths
 
 
-def installRPackages():
+def installRPackages(config: DropConfig = None):
     logger.info("check for missing R packages")
     script = Path(drop.__file__).parent / "installRPackages.R"
     requirements = Path(drop.__file__).parent / 'requirementsR.txt'
 
+    # install main packages
     response = subprocess.run(["Rscript", script, requirements], stderr=subprocess.STDOUT)
     response.check_returncode()
 
+    # install pipeline depending packages
+    if config is not None:
+        pkg_assembly_name = None
+        pkg_mafdb_name = None
+        assemblyID = config.get("genomeAssembly")
+
+        if assemblyID in ['hg19']:
+            pkg_assembly_name = "BSgenome.Hsapiens.UCSC.hg19"
+        elif assemblyID in ['hs37d5']:
+            pkg_assembly_name = "BSgenome.Hsapiens.1000genomes.hs37d5"
+        elif assemblyID in ['hg38']:
+            pkg_assembly_name = "BSgenome.Hsapiens.UCSC.hg38"
+        elif assemblyID in ['GRCh38']:
+            pkg_assembly_name = "BSgenome.Hsapiens.NCBI.GRCh38"
+        else:
+            raise ValueError("Provided genome assembly not known: " + assemblyID)
+
+        if config.get("mae").get("addaf"):
+            if assemblyID in ["hg19", "hs37d5"]:
+                pkg_mafdb_name = "mafdb.gnomad.r2.1.hs37d5"
+            elif assemblyID in ["hg38", "GRCh38"]:
+                pkg_mafdb_name = "MafDb.gnomAD.r2.1.GRCh38"
```

Comment:
Might be nicer to determine `assembly` and `mafdb` package names in `DropConfig` and retrieve them here with a getter function. That function can also be used in the snakemake headers of the R scripts, so that we don't redo the check every time in R.